### PR TITLE
Project2

### PR DESCRIPTION
**Pull request by tongoonamujera**
A Project done on media queries such it fits on different screen sizes that is mobile, desktop and tabklet

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,16 +1,30 @@
...
+![mobile](https://user-images.githubusercontent.com/69446259/109491401-32bd1b00-7a92-11eb-8f1c-294779639314.PNG)
+
+
+SCREENSHOT FOR TABLET VIEW
+![tablet](https://user-images.githubusercontent.com/69446259/109491726-a2cba100-7a92-11eb-860d-1354952800f3.PNG)
+
+
+SCREENSHOT FOR DESKTOP VIEW
+![desktop](https://user-images.githubusercontent.com/69446259/109491990-f5a55880-7a92-11eb-9570-b721ab81483a.PNG)
+
```
Comment (christianotieno) [inline README.md]: ## Improve README
- [ ] I would suggest having just one screenshot of the [page in this project'sREADME`. The more we have the more cluttered the documentation looks. Just a nice short explanation explaining the responsiveness would do in this case. What do you think?

File: README.md [hunk 2]
```diff
@@ -1,16 +1,30 @@
...
+
 Author: Tongoona Mujera
 
+twitter: @tongomujera
+
+github : @tongoonamujera
+
```
Comment (christianotieno) [inline README.md]: ## Improve README
- [ ] Can we make these live links like so 👇 ?
```suggestion
Twitter: [@tongomujera](https://twitter.com/tongoonamujera)

GitHub: [@tongoonamujera](https://github.com/tongoonamujera)
```

File: README.md [hunk 3]
```diff
@@ -1,16 +1,30 @@
...
 Contributing reviews, issues, and feature requests are welcomed!
```
Comment (christianotieno) [inline README.md]: ## Improve the README 
- [ ] Add instructions on how well to clone and run the project locally.
- [ ] You are also welcomed to use this README [template ](https://github.com/microverseinc/readme-template)to act as inspiration on having a more professional looking `README` file as described [here](https://microverse.zendesk.com/hc/en-us/articles/360055660814-Professional-GitHub-repository).

File: index.html [hunk 4]
```diff
@@ -0,0 +1,135 @@
+<!DOCTYPE html>
```
Comment (christianotieno) [inline index.html:1]: ## Improve page layout on large screens 
- We have some elements to take care of in the highlighted sections below have a look
- [ ]  The elements on the page have not been placed as would be expected. We have some unnecessary spacing between the navbar and the three main articles section. The elements in the section below have not been placed as would be expected as well. Hav a look at the comparison below

|[Source Page](https://perma.cc/M5ZV-Q2D6)|[Clone Page](https://raw.githack.com/tongoonamujera/Responsive-Design-MilestoneOne/Project2/index.html)|
|:--:|:--:|
|tnw|tnw-clone|

## Improve layout on small and medium screens
- The changes expected in this breakpoint, mirrors the same ones on the large screen only that we have some additional info to add on this one.
- [ ] Add the `tnw` logo on the page on small and medium screens.
-  [ ] On these small and medium screens, there will be a need to replace the items on the menu in the navbar with the hamburger menu. 
- [ ] Work on the general styling on this section as well. See below the comparisons

|[Source Page](https://perma.cc/M5ZV-Q2D6)|[Clone Page](https://raw.githack.com/tongoonamujera/Responsive-Design-MilestoneOne/Project2/index.html)|
|:--:|:--:|
|tnw-small-medium|tnw-small-medium-clone|

File: style.css [hunk 5]
```diff
@@ -0,0 +1,743 @@
...
+}
+
+.fa-search {
+  padding-top: 10px;
+  border-left: 1px solid gray;
+}
+
+.secnav > nav {
+  margin: 0;
+}
```
Comment (christianotieno) [inline style.css]: ## Aim to remain `DRY`
- [ ] We do have a repeated rule on line `#1 `and `#44` of this file. Do you mind merging them so as to not appear as repeating yourself?

File: style.css [hunk 6]
```diff
@@ -0,0 +1,743 @@
...
+
+.divTwo {
+  background-image: url('images/background3.webp');
+  background-size: 100%;
+}
+
+.divTwo > h1 {
+  padding-top: 10%;
+}
```
Comment (christianotieno) [inline style.css]: ## Aim to remain `DRY`
- [ ] We do have a repeated rule on line `#175 `and `#184` of this file. Do you mind merging them so as to not appear as repeating yourself?

File: style.css [hunk 7]
```diff
@@ -0,0 +1,743 @@
...
+    display: flex;
+    flex-direction: column;
+    width: 100%;
+  }
+
+  .latestFundingTwo {
+    display: flex;
+    flex-direction: column;
+    width: 100%;
+  }
```
Comment (christianotieno) [inline style.css]: ## Aim to remain `DRY`
- [ ] We do have a repeated rule on line `#349` and `#355` of this file. Do you mind merging them so as to not appear as repeating yourself?

File: style.css [hunk 8]
```diff
@@ -0,0 +1,743 @@
...
+
+  .divTwo {
+    background-image: url('images/background3.webp');
+    background-size: 100%;
+  }
+
+  .divTwo > h1 {
+    padding-top: 10%;
+  }
```
Comment (christianotieno) [inline style.css]: ## Aim to remain `DRY`
- [ ] We do have a repeated rule on line `#642` and  `#651` of this file. Do you mind merging them so as to not appear as repeating yourself?

**Review by christianotieno — CHANGES_REQUESTED** [review]
## Changes Required ♻️ 
Hello @tongoonamujera 👋 
The project is coming along just fine. We can make it even much better. See my review below 👇 
### Happy coding 💻 
#### Christian Otieno (CR)

**Review by MuriloRoque — APPROVED** [review]
## Status: Approved :green_circle: 

### Hello @tongoonamujera :wave: 

### Thank you for making all of the requested changes. Your project is approved :clap: :clap: :clap: 

### You may merge, good luck with your next projects :rocket: 

Murilo Roque Paiva da Silva

- Github: [@muriloroque](https://github.com/MuriloRoque)
- Linkedin: [MuriloRoque](https://www.linkedin.com/in/murilo-roque-b1268741/)

[Twitter Follow](https://twitter.com/MuriloRoquePai1)

**Pull request merged**